verification/helpers.go

```diff
@@ -32,6 +70,19 @@ func getArtifactPathFromUri(artifactUri string) (string, error) {
 	return artifactPath, nil
 }
 
+func getArtifactDigestFromUri(artifactUri string) (string, error) {
+	i := strings.LastIndex(artifactUri, ":")
+	if i < 0 {
+		return "", fmt.Errorf("artifact URI %q could not be parsed, make sure it is the fully qualified OCI artifact URI without the scheme/protocol. e.g domain.com:80/my/repository:digest", artifactUri)
+	}
+
+	artifactDigest := artifactUri[i+1:]
+	if artifactDigest == "" {
+		return "", fmt.Errorf("artifact URI %q has an invalid digest %q, make sure the URI is the fully qualified OCI artifact URI without the scheme/protocol. e.g domain.com:80/my/repository:digest", artifactUri, artifactDigest)
+	}
+	return artifactDigest, nil
+}
```

Comment:
It seems this function is not used anywhere. Similarly, other functions like `loadX509TrustStores` are not referenced anywhere. If those function are just for tests, they should be `_test.go` files instead of regular go files.

Reply:
These functions are not for testing. `Verifier` logic will use these. https://github.com/notaryproject/notation-go/blob/main/verification/verifier.go#L21